### ISSUE TITLE
Create png from svg with imagemagick error

### ISSUE THREAD

**Issue by Piioo** [byroot/sprockets-svg#3]
When I call bundle exec rake assets:precompile I got this error:

Magick::ImageMagickError: unable to open image `(null)': No such file or directory @ error/blob.c/OpenBlob/2675
/gems/sprockets-svg-0.4.2/lib/sprockets/svg.rb:14:in`write'

```
def self.convert(svg_path, png_path)
      image = Magick::ImageList.new(svg_path)
      image.write(png_path)
end
```

The funny thing is, that the svg and the png file after the write is there, but ImageMagick throw an error.

sprockets-svg (0.4.2)
rmagick (2.15.0)
imagemagick: stable 6.9.1-1 (bottled), HEAD

**Comment by byroot** [comment]
It happens to me on OSX from time to time, but I never figured the reason why. I assume you are on OSX too?

**Comment by Piioo** [comment]
yes, on OSX, but this happens to a linux debian jessie with imagemagick 6.8 too.
